### DISCUSSION TITLE
Search Engine stops working every 3-4 days

### DISCUSSION THREAD

**Discussion by Bharat-vyas** [requarks/wiki#3283] [Error / Bug Report]
**Describe the bug**
We are using wikijs latest version in docker, with Elasticsearch7 container for search engine functionalities. Everything is working as expected. But after every 3-4 days it is showing some issues with search engine functionality and while searching it is throwing us error "An unexpected error occurred: No pages matching your query." To resolve this we have to rebuild the index in the search engine section.

**To Reproduce**
Search Engine stops working every 3-4 days. We have to rebuild the index manually then it starts working again.

**Host Info (please complete the following information):**
 - OS: [Docker]
 - Wiki.js 2.5.170
 - Database engine: [mysql 5.7]

**Comment by ghost** [comment]
You haven't provided the required info about your host! (OS, Wiki.js version, Database engine)
